getting-started/aggregation.md

```diff
@@ -13,11 +13,18 @@ import CandlestickIntro from "versionContent/_partials/_candlestick_intro.mdx";
 
 # Aggregation
 
-Aggregation refers to a number of different calculations that you can perform on
-your data. For example, if you have data showing temperature changes over time,
-you can calculate an average of those temperatures, or a count of how many
-readings have been taken. Average, sum, and count are all example of simple
-aggregates.
+Aggregation is a way of combing data to get insights from it. At its simplest,
+aggregation is something like looking for an average. For example, if you have
+data showing temperature changes over time, you can calculate an average of
+those temperatures, or a count of how many readings have been taken. Average,
+sum, and count are all example of simple aggregates.
+
+However, aggregation calculations can get complicated, quickly. If you want to
+find the average open and closing values of a range of stocks for each day, then
+you need something a little more sophisticated. That's where Timescale
+continuous aggregates come in.
```

Comment:
Continuous aggregates do not make the aggregation simpler - the query is as complicated as it would be without the cagg. 

They are a performance optimization; they make the queries much faster as you have the aggregation pre-calculated and waiting for you. Even if real time aggregation is enabled, they minimize the number of records fetched from the raw table. It is a mechanism to trade background processing over the span of a time period for blazing fast queries.